phone_number_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative "./phone_number"
+
+RSpec.describe PhoneNumber do
+  let(:phone_number) { PhoneNumber.new("5558675309") }
```

Comment:
`PhoneActions` being called several times here, a better solution to this would be to use 

```suggestion
     before do
       @number = PhoneNumber.new("555-867-5309");
     end
```